src/west/app/config.py

```diff
@@ -48,6 +48,14 @@
 To set a value for <name>, type:
     west config <name> <value>
 
+To append to a value for <name>, type:
+    west config -a <name> <value>
+A value must exist in the selected configuration file in order to be able
+to append to it. The existing value can be empty.
+Examples:
+    west config -a build.cmake-args -- " -DEXTRA_CFLAGS='-Wextra -g0' -DFOO=BAR"
```

Comment:
@marc-hb is this now what you were looking for?

Reply:
LGTM, thanks.